src/components/OrgFinder/FindOrganization.tsx

```diff
@@ -21,30 +30,40 @@ class FindOrganization extends Component<Props, State> {
     this.state = {
       organizations: [],
       displayMap: false,
+      displayError: false,
+      displayIcon: true,
       zipcodeSearch: '',
-      zipcodeLatLng: {},
+      zipcodeLat: {},
+      zipcodeLng: {},
+      zipcodeLatLang: {},
+      results: [],
+      orgsWithinRadius: [],
     };
-    this.getOrganizations = this.getOrganizations.bind(this);
-    this.geocodeZipcode = this.geocodeZipcode.bind(this);
+    this.getAllOrganizations = this.getAllOrganizations.bind(this);
+    // this.geocodeZipcode = this.geocodeZipcode.bind(this);
     this.onHandleChangeZipcode = this.onHandleChangeZipcode.bind(this);
     this.onSubmitZipcode = this.onSubmitZipcode.bind(this);
   }
 
   componentDidMount() {
-    this.getOrganizations();
+    this.getAllOrganizations();
   }
 
-  getOrganizations() {
+  getAllOrganizations() {
+    const count = 0;
```

Comment:
can you write the fetch call? The backend route is called: 
/get-all-orgs

Takes in a json object specifying usertypes and orgnames
{userTypes : [],
 organizations : []}

https://github.com/keepid/keepid_server/commit/8f082584fed8ac346f353b9d1bdc0bcae2083fdf